README.md

```diff
@@ -1,41 +1,49 @@
-# Build environment for Rust crates
+# Build environment for third-party Rust crates
 
-This repository contains the source of the Docker container the Rust project
-uses to build third-party crates. It is based on **Ubuntu 20.04**, and contains
-all the native dependencies used by the Rust crates we know of.
+This repository contains the source code and the tooling to produce the Docker
+containers used by [Crater] and [docs.rs] to build third-party crates. The
+contents of this repository are released under the MIT license.
 
 ## Adding new dependencies
 
-If your crate fails to build on one of the services that uses this Docker
-image, please either open an issue with the name of the packages you need or
-send a pull request that adds the packages to `packages.txt` or
-`packages-backports.txt` (if the package is in the `stretch-backports` suite).
+If your crate fails to build on [Crater] or [docs.rs], you can:
 
-## Using the Docker image
+* [Open an issue][new-issue-linux] with the names of the packages you need
+* Send a PR adding the package names to the `linux/packages.txt` file
 
-The Docker image is automatically built after a commit is pushed to master, and
-it's available [on Docker Hub][dockerhub] as `rustops/crates-build-env`. You
-can get it with:
+Note that the package needs to be available in the **Ubuntu 20.04** archives.
+
+## Available containers
+
+### `linux`
+
+This container is based on **Ubuntu 20.04** and includes all the native
+dependencies used by Rust crates we know of. It's used as the build environment
+for the [Crater] and [docs.rs] projects.
+
+You can pull this container by running:
 
 ```
-$ docker pull rustops/crates-build-env
+docker pull ghcr.io/rust-lang/crates-build-env/linux:latest
 ```
 
-The image **does not** contain a Rust toolchain in it: you need to manually
-mount the toolchain you want to use inside the container.
+### `linux-micro`
 
-### Mapping the user id between the container and the system
+This container is based on **Ubuntu 20.04** and includes the minimum set of
+dependencies needed to compile simple Rust programs. It's used by the test
+suites of [Crater] and [docs.rs] and during local development.
 
-By default, the user id inside a Docker container is `0` (root). That doesn't
-cause any security risk thanks to the container isolation, but it might pose
-some problems when the container writes into directories mounted from the host,
-since all the files in those directories will be owned by root.
```

Comment:
These two paragraphs (that there's no toolchain mounted and that the user ID is 0) got deleted. I think we should at least keep the first one - is the second still relevant now that MAP_USER_ID doesn't exist?

Reply:
Oh, I accidentally removed the first paragraph, good catch! The second one is not really relevant anymore.